pkgs/data/fonts/iosevka/default.nix

```diff
@@ -1,24 +1,45 @@
-{ stdenv, lib, pkgs, fetchFromGitHub, nodejs, nodePackages, ttfautohint-nox
+{ stdenv, lib, pkgs, fetchFromGitHub, nodejs, nodePackages, remarshal
+, ttfautohint-nox
   # Custom font set options.
   # https://typeof.net/Iosevka/customizer
-  #
+
+  # Can be a raw toml string, or a nix config.
```

Comment:
Nit:

```suggestion
  # Can be a raw TOML string, or a Nix attrset.
```

Reply:
Fair enough! Looks like I also have to fix node conflicts, so I'll do both in one go.